monitor.py

```diff
@@ -173,12 +173,12 @@ def run_command(args, soft_time_limit=15, hard_time_limit=60, include_output=Tru
 
     # Add to the list
     duration = end_time - start_time
-    l = durations[str(args)]
-    l.append(duration)
-    while len(l) > 1000:  # Keep the list bounded
-        l.pop(0)
-    average_duration = sum(l) // len(l)
-    max_duration = max(l)
+    list_of_args = durations[str(args)]
```

Comment:
It looks like key is the args and the value is the durations. Can you rename `list_of_args` to `durations` and `durations` to `args_to_durations`?